docs/reference/objects/product/variant/experience_variant/index.md

```diff
@@ -18,9 +18,23 @@ it has access to the following attributes.
 
 #### Attributes
 
+## `experience_variant.payment_plan`
+{: .d-inline-block }
+[Payment Plan]({% link docs/reference/objects/product/variant/payment_plan.md %})
+{: .label .fs-1 }
+
+The same as [variant]({% link
+docs/reference/objects/product/variant/index.md %}) `deposit` method
+with the caveat that if the experience variant is accessed through an
+[experience slot]({% link docs/reference/objects/product/experience_slot.md
+%}), a [payment plan]({% link
+docs/reference/objects/product/variant/payment_plan.md %}) will only be
```

Comment:
I think we should keep the reference to the payment plan separate (in the same way it is on the previous variant page). In that case, do we need to split this out from the normal variant.deposit, if they return the same information? Or have i got the wrong end of the stick here?

Reply:
Hey @samantha-lind can you clarify what you mean by split out here?

Reply:
Sorry @benlavon , wasn't super clear. I mean do we need to add this extra clarification for experience_variant.deposit, if variant.deposit works in the same way. Or have I misunderstood? From what I understand, it's just the payment plan bit that is different?

Reply:
I've added a separate commit to document how `deposit` works now if you have a sec to to take another look?

Reply:
@benlavon @samantha-lind is OOO for the next week, but I've reviewed and this change looks good/seems to solve her query.